tests/helpers/repo_utils.py

```diff
@@ -0,0 +1,39 @@
+import os
+import stat
+
+from git import Repo
+
+
+def create_clean_repo():
+    repo = Repo.init()
+    repo.create_remote("origin", "https://github.com/user/repo.git")
+    repo.config_writer().set_value("user", "name", "test").release()
+    repo.config_writer().set_value("user", "email", "test@test.io").release()
+    with open("clean.txt", "w") as fp:
+        fp.close()
```

Comment:
redundant

Reply:
which part ?
I have to configure these for git otherwise it throws an error on CI